config/webpack/clientWebpackConfig.js

```
@@ -1,20 +1,11 @@
-// The source code including full typescript support is available at:
-// https://github.com/shakacode/react_on_rails_tutorial_with_ssr_and_hmr_fast_refresh/blob/master/config/webpack/clientWebpackConfig.js
+// The source code including full typescript support is available at: 
```

ESLint

[error] Delete `·`

(prettier/prettier)